modules/loader/src/main/java/com/opengamma/strata/loader/csv/FxSingleTradeCsvPlugin.java

```diff
@@ -42,33 +49,23 @@
 /**
  * Handles the CSV file format for FX Single trades.
  */
-class FxSingleTradeCsvPlugin implements TradeCsvParserPlugin, TradeTypeCsvWriter<FxSingleTrade> {
+class FxSingleTradeCsvPlugin implements TradeCsvParserPlugin, TradeCsvWriterPlugin<FxSingleTrade> {
 
   /**
    * The singleton instance of the plugin.
    */
   public static final FxSingleTradeCsvPlugin INSTANCE = new FxSingleTradeCsvPlugin();
 
-  static final String LEG_1_DIRECTION_FIELD = "Leg 1 " + DIRECTION_FIELD;
-  static final String LEG_1_PAYMENT_DATE_FIELD = "Leg 1 " + PAYMENT_DATE_FIELD;
-  static final String LEG_1_CURRENCY_FIELD = "Leg 1 " + CURRENCY_FIELD;
-  static final String LEG_1_NOTIONAL_FIELD = "Leg 1 " + NOTIONAL_FIELD;
-
-  static final String LEG_2_DIRECTION_FIELD = "Leg 2 " + DIRECTION_FIELD;
-  static final String LEG_2_PAYMENT_DATE_FIELD = "Leg 2 " + PAYMENT_DATE_FIELD;
-  static final String LEG_2_CURRENCY_FIELD = "Leg 2 " + CURRENCY_FIELD;
-  static final String LEG_2_NOTIONAL_FIELD = "Leg 2 " + NOTIONAL_FIELD;
-
   /** The headers. */
-  private static final ImmutableList<String> HEADERS = ImmutableList.<String>builder()
-      .add(LEG_1_DIRECTION_FIELD)
-      .add(LEG_1_PAYMENT_DATE_FIELD)
-      .add(LEG_1_CURRENCY_FIELD)
-      .add(LEG_1_NOTIONAL_FIELD)
-      .add(LEG_2_DIRECTION_FIELD)
-      .add(LEG_2_PAYMENT_DATE_FIELD)
-      .add(LEG_2_CURRENCY_FIELD)
-      .add(LEG_2_NOTIONAL_FIELD)
+  private static final Set<String> HEADERS = ImmutableSet.<String>builder()
+      .add(CsvLoaderColumns.LEG_1_DIRECTION_FIELD)
```

Comment:
`CsvLoaderColumns` should be statically imported in this class (multiple occurrences

Reply:
Done, thanks.